architecture/service_catalog/aws_broker.adoc

```diff
@@ -0,0 +1,73 @@
+[[admin-guide-get-started-aws]]
+= {product-title} Amazon Web Services Broker
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+The _AWS Service Broker_ provides access to Amazon Web Services (AWS) through the {product-title} Service Catalog. 
+
+[NOTE]
+====
+The AWS Service Broker is supported and qualified on OpenShift Container Platform.  It is offered directly from Amazon as a download and as such, many components of this service broker solution are supported directly by Amazon. Red Hat provides support for the installation of the AWS Service Broker and troubleshooting of the OpenShift cluster and Service Catalog issues.
+====
+
+This guide describes how to deploy the AWS Service Broker into your {product-title} cluster.
+
+After completing the steps in this guide, the following AWS services are available from the OpenShift Service Catalog:
+
+* link:https://aws.amazon.com/sqs/[Simple Queue Service (SQS)]
+* link:https://aws.amazon.com/sns/[Simple Notification Service (SNS)]
+* link:https://aws.amazon.com/route53/[Route 53]
+* link:https://aws.amazon.com/rds/[Relational Database Service (RDS)]
+* link:https://aws.amazon.com/emr/[Elastic MapReduce (EMR)]
+* link:https://aws.amazon.com/s3/[Simple Cloud Storage Service (S3)]
+* link:https://aws.amazon.com/elasticache/[ElastiCache]
+* link:https://aws.amazon.com/redshift/[Redshift]
+* link:https://aws.amazon.com/dynamodb/[DynamoDB]
+* link:https://aws.amazon.com/athena/[Athena]
+
+Before working with this guide, familiarize yourself with the following terms:
+
+{product-title} Service Catalog::
+The _Service Catalog_ is the {product-title} component that finds and presents the services a user has access to. It also gives that user the ability to provision new instances of those services and provide a way to bind the provisioned services to existing applications.
+
+AWS Service Brokers:: 
+The link:https://aws.amazon.com/partners/servicebroker/[_AWS Service Broker_] is an implementation of the link:https://www.openservicebrokerapi.org/[Open Service Broker API] to publish services to external environments, such as Kubernetes and the {product-title} Service Catalog.
```

Comment:
{product-title}  or OpenShift? Please check and be consistent. (You used OpenShift earlier.)